working/templates/srmfunction/Profile Data/Definitions/readme.md

```diff
@@ -0,0 +1 @@
+This folder contains Profile Definition JSON files associated with this SRM Function solution.
```

Comment:
consider renaming the readme files to e.g. about.md (to be in line with other templates we are working on)